docs/sources/tools/logcli.md

```diff
@@ -282,101 +278,118 @@ Example:
+      --key=""                   Path to the client certificate key. Can also be set using LOKI_CLIENT_KEY_PATH env var.
+      --org-id=""                adds X-Scope-OrgID to API requests for representing tenant ID. Useful for requesting tenant data when
+                                 bypassing an auth gateway. Can also be set using LOKI_ORG_ID env var.
+      --query-tags=""            adds X-Query-Tags http header to API requests. This header value will be part of `metrics.go` statistics.
+                                 Useful for tracking the query. Can also be set using LOKI_QUERY_TAGS env var.
+      --bearer-token=""          adds the Authorization header to API requests for authentication purposes. Can also be set using
+                                 LOKI_BEARER_TOKEN env var.
+      --bearer-token-file=""     adds the Authorization header to API requests for authentication purposes. Can also be set using
+                                 LOKI_BEARER_TOKEN_FILE env var.
+      --retries=0                How many times to retry each query when getting an error response from Loki. Can also be set using
+                                 LOKI_CLIENT_RETRIES env var.
+      --min-backoff=0            Minimum backoff time between retries. Can also be set using LOKI_CLIENT_MIN_BACKOFF env var.
+      --max-backoff=0            Maximum backoff time between retries. Can also be set using LOKI_CLIENT_MAX_BACKOFF env var.
+      --auth-header="Authorization"
+                                 The authorization header used. Can also be set using LOKI_AUTH_HEADER env var.
+      --proxy-url=""             The http or https proxy to use when making requests. Can also be set using LOKI_HTTP_PROXY_URL env var.
+      --limit=30                 Limit on number of entries to print.
+      --since=1h                 Lookback window.
+      --from=FROM                Start looking for logs at this absolute time (inclusive)
+      --to=TO                    Stop looking for logs at this absolute time (exclusive)
+      --step=STEP                Query resolution step width, for metric queries. Evaluate the query at the specified step over the time
+                                 range.
+      --interval=INTERVAL        Query interval, for log queries. Return entries at the specified interval, ignoring those between. **This
+                                 parameter is experimental, please see Issue 1779**
+      --batch=1000               Query batch size to use until 'limit' is reached
+      --parallel-duration=1h     Split the range into jobs of this length to download the logs in parallel. This will result in the logs
+                                 being out of order. Use --part-prefix to create a file per job to maintain ordering.
+      --parallel-max-workers=1   Max number of workers to start up for parallel jobs. A value of 1 will not create any parallel workers.
```

Comment:
Question - What happens with a value of 0?

Reply:
It would not start up any workers, and exit. I guess it would be pretty stupid to set this value. I will see if there's any validation done on the parameters and add some error messages there if such a value is set.